examples/commercial-quick-order/.env.template

```diff
@@ -0,0 +1,4 @@
+VITE_SHOPWARE_API="https://201023-quick-order.swstage.store/"
```

Comment:
"Version is flagged for deletion".
Adding a temporary instance to the example is not a good idea. We should use our demo or "force" users to add their own instance.

Reply:
thanks for pointing that! I changed the instance's URL